projects/docs-tools/src/modules/demo-page/demo-page.component.ts

```diff
@@ -54,7 +56,27 @@ export class SkyDocsDemoPageComponent
    * Specifies the local path to any other relevant source code. The values are relative to the root directory.
    */
   @Input()
-  public additionalSourceCodePaths: string[];
+  public set additionalSourceCodePaths(paths: string[] | undefined) {
+    this.#_additionalSourceCodePaths = paths;
+    this.#updateTypes(true);
```

Comment:
It's not immediately obvious what passing `true` to this method does. I would separate the method into two to make it more clear.
```
this.#updateTypes();
this.#updateTestingTypes();
```

Reply:
Another option is to always update both sets of types in the `#updateTypes` method (and exclude the boolean parameter altogether).

Reply:
I split these into two different functions